templates/patterns/navigation_menu/pattern-navigation-menu.html.twig

```diff
@@ -46,11 +46,5 @@
     items: _items,
     icon_path: ecl_icon_path
   } only %}
-{% elseif site_name %}
-  <div class="ecl-site-header-{{ ecl_branding }}__banner">
-    <div class="ecl-container">
-      {{ site_name }}
-    </div>
-  </div>
```

Comment:
We need to remove this from the pattern definition too. Let's just remove it and add a note in the readme about it. Before that, please make sure that, on mobile, it all still behaves correctly.

Reply:
Not really, the navigation menu still needs a site name variable since it is still used in mobile.

Reply:
Ah right, sorry, my bad!